main.py

```diff
@@ -91,15 +102,15 @@ def mean_reversion():
     main function to be executed
     """
     while True:
-        if TradingVenue().check_if_open():
+        if client.market_data.venues.get(os.getenv('MIC')).results[0].is_open:
             # make buy or sell decision
             check_if_buy(
-                isin="US88160R1014",  # this is Tesla, but you can obviously use any ISIN you like :)
+                isin="DE0007664039",  # this is Tesla, but you can obviously use any ISIN you like :)
                 x1="d1"
             )
         else:
             # sleep until market reopens in case it is closed
-            time.sleep(TradingVenue().seconds_till_tv_opens())
+            time.sleep(client.market_data.venues.get('xmun'))
```

Comment:
is this correct? Seems like you need to stick with the initial function to receive the correct number of seconds until the venue reopens. Especially because you modified the function in the TradingVenue.py file to match with the new SDK.

Reply:
My bad, fixed it.